wiki/People/The_Team/en.md

```diff
@@ -15,13 +15,24 @@ Active
 
 | Name | Description |
 | ---- | ----------- |
+| ![][flag_US] [Chaos](https://osu.ppy.sh/users/2628870) | Moderation and community advisor |
 | ![][flag_NO] [cYsmix](https://osu.ppy.sh/users/272870) | Audio design & [official osu! Featured Artist](https://osu.ppy.sh/beatmaps/artists/2) |
-| ![][flag_AU] [Ephemeral](https://osu.ppy.sh/users/102335) | Media, Featured Artist outreach, Meganekko secretary |
+| ![][flag_NZ] [deadbeat](https://osu.ppy.sh/users/128370) | World Cup organization and administration |
+| ![][flag_AU] [Ephemeral](https://osu.ppy.sh/users/102335) | Project/community maintenance and Featured Artist outreach |
 | ![][flag_JP] [flyte](https://osu.ppy.sh/users/3103765) | Lead designer, [osu!next](https://next.ppy.sh/) |
+| ![][flag_US] [HappyStick](https://osu.ppy.sh/users/256802) | osu! coffee hour host, World Cup streaming and tournament organization |
```

Comment:
`osu! coffee hour` -> `osu! Coffee Hour`
`organization` -> `organisation` Do it for every instance.